projects/js-packages/components/components/action-popover/stories/styles.module.scss

```diff
@@ -0,0 +1,11 @@
+.wrapper {
+	height: 100vh;
+	width: 100%;
```

Comment:
`100vw`?

Reply:
Not necessary. Removed the width, as it is already all the available space.